presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -95,7 +96,8 @@ public Environment start()
     private Environment tryStart()
     {
         try {
-            pruneEnvironment();
+            // We need to prune containers only as we are reusing existing network
```

Comment:
am i right the network is a static singleton?

Reply:
nope, it's a field in a Builder, as we are building Environment once and then starting to start it multiple times, when we remove network, containers won't be able to start.

Reply:
```
@SuppressWarnings("resource")
        private Network network = Network.builder()
                .createNetworkCmdModifier(createNetworkCmd ->
                        createNetworkCmd
                                .withName(PRODUCT_TEST_LAUNCHER_NETWORK)
                                .withLabels(ImmutableMap.of(PRODUCT_TEST_LAUNCHER_STARTED_LABEL_NAME, PRODUCT_TEST_LAUNCHER_STARTED_LABEL_VALUE)))
                .build();
```